com.brave.Brave.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
```

Comment:
I would give it a shot with `20.08` and see what happens :thinking:

Reply:
I honestly don't know for sure if this is a good idea. I wanted to do that but https://github.com/refi64/zypak says I need to use 19.08

From https://github.com/refi64/zypak :
> This requires your Flatpak to be using:
> 
>     org.freedesktop.Platform / Sdk version 19.08.
>     org.electronjs.Electron2.BaseApp as your base app. Recent releases include Zypak built-in.
> 
> Now, instead of running your Electron binary directly, call it via zypak-wrapper PATH/TO/MY/ELECTRON/BINARY.

Reply:
Can you check if it works properly with 20.08?

Reply:
20.08 is still building. I will push if it works

Reply:
I think it should. Version `20.08` was updated recently and that's what I'm using in Wire. Let me try.

Reply:
It worked for me. Tell me if you notice anything wrong. Also, I appreciate if you could help with the printing problem (note: `--sockets=cups` doesn't help)